core/src/main/scala/kafka/log/Log.scala

```diff
@@ -1144,7 +1144,7 @@ class Log(@volatile var dir: File,
   /**
    * The size of the log in bytes
    */
-  def size: Long = logSegments.map(_.size).sum
+  def size: Long = logSegments.map(_.size).foldLeft(0L)(_ + _)
```

Comment:
yes, not sure which one is better though. Yours might be more intuitive I guess.